packages/atomic/src/components.d.ts

```diff
@@ -6039,6 +6056,16 @@ declare namespace LocalJSX {
          */
         "store"?: AtomicCommonStore<AtomicCommonStoreData>;
     }
+    interface AtomicProductDescription {
+        /**
+          * Which description field to use
+         */
+        "field"?: 'ec_description' | 'ec_shortdesc';
+        /**
+          * The number of lines after which the product description should be truncated. A value of -1 will disable truncation
+         */
+        "truncateAfter"?: '-1' | '1' | '2' | '3' | '4';
```

Comment:
Tailwind offers `line-clamp-${n}` from 1 to 6, and `none` when there is no line clamp. Should we do the same, instead of using `-1`, or is there a standard in atomic?

https://tailwindcss.com/docs/line-clamp

Reply:
I don't think we need to offer up to line-clamp 6, but I do think it makes sense to replace "-1" with something more verbose, since we don't use the prop value as a number anyway